packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts

```diff
@@ -3,8 +3,10 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why was this needed?

Reply:
Good question! I added `lodash.groupby` to the package dependencies instead 👍